pyproject.toml

```diff
@@ -49,6 +49,8 @@ toml = {version=">=0.10.2", optional=true}
 tzlocal = "^5.0"
 python-decouple = ">=3.8"
 click = ">=8.0.0"
+nptdms = "^1.9.0"
```

Comment:
Use `>=` unless you have reason to believe 2.0 will break compatibility

Reply:
Using `^` means "I don't want to automatically update to 2.0".

It is unfortunate that `poetry add` uses `^` by default.